i18n/de.po

```diff
@@ -1412,6 +1571,14 @@ msgstr "Origin"
 msgid "Origin is EA's PC games portal."
 msgstr "Origin ist EA's Portal für PC-Spiele."
 
+#: i18n/tmp/Applications/Games/Origin/Local (Legacy)/script.js
+msgid "Local (Legacy)"
+msgstr "Lokal (Legacy)"
```

Comment:
Maybe better `Lokal (Alt)`?

Reply:
I'm not sure. I think "Legacy" is the wording which is used officially by EA.